.github/workflows/local-dev-ci.yaml

```diff
@@ -0,0 +1,60 @@
+name: Execute local test
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+    paths-ignore:
+      - "docs/**"
+      - "**/*.md"
+
+jobs:
+  local-dev-test:
+    name: local-dev-test
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/setup-go@v3
+        with:
+          go-version: v1.17
+
+      - uses: azure/setup-kubectl@v3
```

Comment:
out of curiosity why setup-kubectl rather than kubectl-simple or kube-tools?

Reply:
also looking further down, isn't [kind-action already bringing kubectl](https://github.com/container-tools/kind-action/blob/main/kind.sh#L149-L154)?

Reply:
you are right, kind-action already bring kubectl. fixed